kmir/src/kmir/kdist/mir-semantics/rt/data.md

```diff
@@ -581,6 +581,58 @@ The solution is to use rewrite operations in a downward pass through the project
      andBool (FORCE orBool MUT ==K mutabilityMut)
     [preserves-definedness]
 
+  rule <k> #projectedUpdate(
+            _DEST,
+            typedValue(Ptr(OFFSET, place(LOCAL, PLACEPROJ), MUT, _ADDRESS, _OFFSET2), _, _),
+            projectionElemDeref PROJS,
```

Comment:
There is also `#readProjection` earlier in this module, which will require the same kind of extension to `Deref` a pointer.
https://github.com/runtimeverification/mir-semantics/blob/1ee2475e10c728326275dce14748562eaee81d7f/kmir/src/kmir/kdist/mir-semantics/rt/data.md?plain=1#L314-L322